SECURITY.md

```diff
@@ -0,0 +1,14 @@
+Reporting Security Issues
+------------------
+
+If you think you have found a security vulnerability, please send a report to [security@grafana.com](mailto:security@grafana.com). This address can be used for all of
```

Comment:
Should there be something about not disclosing the vulnerability before we fix it or give permission to disclose it?

Reply:
Good point. Pushed a few fixes.